Sources/ForageSDK/Foundation/Protocol/ForagePANElement.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  ForagePANElement.swift
+//
+//
+//  Created by Danny Leiser on 11/23/23.
+//  Copyright © 2023-Present Forage Technology Corporation. All rights reserved.
+//
+
+/// Added information that can be inferred from the card value.
+public protocol CardInformation {
+    var binValue: String? { get }
```

Comment:
@djoksimo @devinmorgan would love input on what we should call this field. What would be a good name that could correspond to other information for PayFac for example? I don't want to make it specific to EBT cards and I also want to avoid using the word "state" if possible just because that feels confusing.

Reply:
Thank you for opening this discussion.

> I don't want to make it specific to EBT card

I think the derived US state will be EBT-specific and it's safe to add a note like "This value is only set (non-nil) for EBT cards."

----

- I think it's safe to drop the `Value` and `Information` suffixes
- `Card` and `iin` align with conventions seen in other payments libraries.
- IIN is used more commonly throughout our client libraries and I believe we generally opt for "IIN" > "BIN"
- I've added a proposed suggestion for the `Card`'s comment
- I've added a comment for the `iin` property
- I think we should expose a public `USState` enum as seen below 👇🏻  (this may involve refactoring the `ForagePANValidations.swift` class

```suggestion
/// Represents a state of the United States of America.
public USState: String {
    case alabama = "AL"
    case alaska = "AK"
    case arizona = "AZ"
    case arkansas = "AR"
    ....
}

/// Represents card information based on the user's Primary Account Number (PAN) input.
public protocol Card {
    /// ------ IF WE WANT TO/CAN EXPOSE THE IIN to the client, other libraries do this for credit cards --------
    /// An acronym for Issuer Identification Number (IIN), also known as BIN (Bank Identification Number).
    /// The IIN is the first 6 digits of the PAN.
    iin: String? { get }
    
    /// The US state that issued the EBT card, derived from the Issuer Identification Number (IIN), 
    /// also known as BIN (Bank Identification Number).
    /// The IIN is the first 6 digits of the PAN.
    var usState: USState? { get }
```

Thoughts?

Reply:
Trying to adopt an MVP stance on this, what do you guys think about just exposing the `iin: String` and not exposing the `usState: USState` for now, at least? Reasons to consider this:

1. Telling GoPuff the card's IIN would be sufficient for them to gracefully navigate the Massachusetts issues since they could choose to handle cards with Massachusetts' IIN (`600875`) how they wish
2. We can always decide to add in `usState` later on if `iin` is found to be sufficiently useful.
3. My understanding is that IIN is relevant beyond just the scope of EBT cards. So, as we add more payment methods, IIN may continue to be relevant whereas (again just my understanding) `usState` may be EBT / benefits specific (e.g. `usState` doesn't matter if Forage ever processes credit cards in house)

What's your guys' take?